### PR TITLE
README: Add coding style

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,35 @@ If you have questions, visit us on irc.freenode.net in #opensuse-factory or
...
 
 
+### Coding style
+
+The project follows the rules of the parent project
+[os-autoinst](https://github.com/os-autoinst/os-autoinst#how-to-contribute).
+and additionally the following rules:
+
+* Take
+  [example boot.pm](https://github.com/os-autoinst/os-autoinst-distri-example/blob/master/tests/boot.pm)
```
Comment (asmorodskyi) [inline README.md]: not good idea , need to point to something static here . you can't guarantee that after some time someone will not introduce some anti-pattern into this file. Copy-paste file to README ?

Comment (coolo) [inline README.md]: you can also link revisions if you're paranoid

Comment (foursixnine) [inline README.md]: linking against master's version LGTM, as that will usually guarantee we have the latest code as reference. Linking to a specific version is not needed at this point in time.

Comment (asmorodskyi) [inline README.md]: LGTM :)

**Pull request merged**